lib/internal/process/esm_loader.js

```diff
@@ -79,20 +106,21 @@ function loadModulesInIsolation(specifiers, loaders = []) {
   try {
     cwd = process.cwd() + '/';
   } catch {
-    cwd = 'file:///';
+    cwd = '/';
   }
```

Comment:
Could this possibly change between `initializeLoader()` and here? If not, I think `parentURL` should be passed as an argument to `loadModulesInIsolation(specifiers, parentURL, loaders = [])`.

Reply:
Fixed 👍